lib/condor.py

```diff
@@ -300,7 +321,14 @@ def submit(
     # correct vault token storer script
     # In the condor_vault_storer output, debug gives us more output than verbose,
     # so make that mapping from our verbose/output to condor_vault_storer's
-    _sec_cred_storer_val = f"{jldir}/bin/condor_vault_storer"
+
+    if vargs["managed_token"] and ran_vault_storer_recently(
+        schedd_name, vargs["oauth_handle"], vargs["outbase"]
+    ):
+        _sec_cred_storer_val = "/bin/true"
```

Comment:
Rather than hardcode `/bin/true` here and below, could we set a global constant called `NO_OP_STORER` or something like that, and set that to `/bin/true`?

Then this line would be very clear that: `_sec_cred_storer_val = NO_OP_STORER`.

Reply:
Done.